### PR TITLE
change share sign in with mobile apps guide to single page

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** Updated https://developer.okta.com/docs/guides/shared-sso-android-ios/android/overview/ to single page guide
- **Is this PR related to a Monolith release?** No
### Resolves:

* [OKTA-438729](https://oktainc.atlassian.net/browse/OKTA-438729)

### CODE REVIEW

File: packages/@okta/vuepress-site/conductor.yml [hunk 1]
```diff
@@ -2583,6 +2583,90 @@ redirects:
     to: /docs/guides/common-hook-set-up-steps/-/main/#troubleshoot-hook-implementations
   - from: /docs/guides/unlock-mobile-app-with-biometrics/ios/next-steps/index.html
     to: /docs/guides/unlock-mobile-app-with-biometrics/ios/main/
+  - from: /docs/guides/shared-sso-android-ios/-/overview
+    to: /docs/guides/shared-sso-android-ios/-/main
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/conductor.yml:2587]:
```suggestion
    to: /docs/guides/shared-sso-android-ios/
```

File: packages/@okta/vuepress-site/conductor.yml [hunk 2]
```diff
@@ -2583,6 +2583,90 @@ redirects:
...
+  - from: /docs/guides/shared-sso-android-ios/android/setup-first-mobile-app/index.html
+    to: /docs/guides/shared-sso-android-ios/android/main/#set-up-the-first-mobile-app
+  - from: /docs/guides/shared-sso-android-ios/android/create-second-mobile-app/index.html
+    to: /docs/guides/shared-sso-android-ios/android/main/#create-a-second-mobile-app
+  - from: /docs/guides/shared-sso-android-ios/android/optional-settings/index.html
+    to: /docs/guides/shared-sso-android-ios/android/main/#optional-settings
+  - from: /docs/guides/shared-sso-android-ios/android/next-steps/index.html
+    to: /docs/guides/shared-sso-android-ios/android/main/#next-steps
+  - from: /docs/guides/shared-sso-android-ios/ios/overview
+    to: /docs/guides/shared-sso-android-ios/ios/main
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/conductor.yml:2643]:
```suggestion
    to: /docs/guides/shared-sso-android-ios/ios/main/
```

File: packages/@okta/vuepress-site/conductor.yml [hunk 3]
```diff
@@ -2583,6 +2583,90 @@ redirects:
...
+  - from: /docs/guides/shared-sso-android-ios/ios/setup-first-mobile-app
+    to: /docs/guides/shared-sso-android-ios/ios/main/#set-up-the-first-mobile-app
+  - from: /docs/guides/shared-sso-android-ios/ios/create-second-mobile-app
+    to: /docs/guides/shared-sso-android-ios/ios/main/#create-a-second-mobile-app
+  - from: /docs/guides/shared-sso-android-ios/ios/optional-settings
+    to: /docs/guides/shared-sso-android-ios/ios/main/#optional-settings
+  - from: /docs/guides/shared-sso-android-ios/ios/next-steps
+    to: /docs/guides/shared-sso-android-ios/ios/main/#next-steps
+  - from: /docs/guides/shared-sso-android-ios/ios/overview/index.html
+    to: /docs/guides/shared-sso-android-ios/ios/main
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/conductor.yml:2657]:
```suggestion
    to: /docs/guides/shared-sso-android-ios/ios/main/
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+**What you need**
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup)
+* Android Studio with an emulator for Android testing
+* Xcode with a simulator for iOS testing
+
+**Sample code**
+
+See the [Sample code](#sample-code) section.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:26]: Re-do **Sample code** section based on our discussion with stack-selectors and nutrition fact template?

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+## Session and persistent Single Sign-On
+
+Single Sign-On (SSO) allows users to authenticate once and access multiple resources without being prompted for additional credentials. Okta supports both session and persistent SSO:
+
+* **Session SSO**: Session SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications during a particular session. However, if a particular session ends, the user is prompted for their credentials again.
+* **Persistent SSO**: Persistent SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications for as long as the persistent SSO cookie is valid.
+
+The difference between persistent SSO and session SSO is that persistent SSO can be maintained across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session with native mobile apps, you need to enable persistent SSO. 
+
+<StackSelector snippet="enablesso" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:43]:
```suggestion
<StackSelector snippet="enablesso" noSelector />
```
If we have a stack selector at the top of the page, then we don't need to show one here.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+Next, you set up the mobile applications using the configuration from these native apps that you just created.
+
+## Set up the first mobile app
+
+In this section, you configure settings for the first mobile app.
+
+> Note: This section assumes that you have already downloaded the appropriate sample apps — see the sample links at the top of the article.
+
+<StackSelector snippet="configfile" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:72]:
```suggestion
<StackSelector snippet="configfile" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### Add the redirect scheme
+
+To redirect back to your application from a web browser, specify a unique URI to your app.
+
+<StackSelector snippet="addredirectscheme" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:78]:
```suggestion
<StackSelector snippet="addredirectscheme" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+## Create a second mobile app
+
+You need a second mobile app to test with.
+
+<StackSelector snippet="createsecondapp" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:84]:
```suggestion
<StackSelector snippet="createsecondapp" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+## Optional settings
+
+There are a few additional settings that you can play with while testing shared SSO that involve the use of the `prompt` parameter. See [Parameter details](https://developer.okta.com/docs/reference/api/oidc/#parameter-details) for more information on using the `prompt` parameter.
+
+### Always prompt for sign in regardless of session
+If you are using the same Okta domain for both of your apps, the default behavior when a session already exists is that the user is silently authenticated without a sign-in prompt. If your second application requires a prompt for sign-in regardless of session, you can configure this by passing in the `prompt=login` parameter.
+
+<StackSelector snippet="promptsignin" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:93]:
```suggestion
<StackSelector snippet="promptsignin" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+### Check for a valid session
+
+You can also check if the browser has a valid session by using the `prompt=none` parameter. The `prompt=none` parameter guarantees that the user isn't prompted for credentials. Either the requested tokens are obtained or if the session is invalid or doesn't exist, the application receives an OAuth error response. See [Parameter details](https://developer.okta.com/docs/reference/api/oidc/#parameter-details) for more information on using the `prompt` parameter.
+
+If your application requires that the user signs in to the first app first, then you can use the `prompt=none` parameter in the second app to check whether the user is already signed in to the first app. 
+
+<StackSelector snippet="checkvalidsession" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:101]:
```suggestion
<StackSelector snippet="checkvalidsession" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+
+### Clear the session
+
+To clear a session, add the following code to both of your apps:
+
+<StackSelector snippet="clearsession" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:107]:
```suggestion
<StackSelector snippet="clearsession" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+You should now have a better understanding of how a sign-in session is shared between two mobile apps on a device and the knowledge to configure your own apps.
+
+To learn more about our Mobile OpenID Connect (OIDC) SDKs and sample apps:
+
+* [Okta OIDC for iOS](https://github.com/okta/okta-oidc-ios)
+* [Okta OIDC for Android](https://github.com/okta/okta-oidc-android)
+* [Okta iOS Browser Sign-in Sample App](https://github.com/okta/samples-ios/tree/master/browser-sign-in)
+* [Kotlin Sign-In Sample App for Android](https://github.com/okta/samples-android/tree/master/sign-in-kotlin)
+
+## Sample code
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:120]: Convert this to stack-snippet in the nutrition facts?

**Review by vanngo-okta — COMMENTED** [review]
The redirects look good!
A few nits regarding the `noSelector` option. Also need to convert the Sample code section to stack-snippet in nutrition facts based on the updated template?

**Comment by chrisdmills-okta** [comment]
Super, thanks so much for all the fixes, @vanngo-okta . I've committed all of them, and then updated the sample code stack selector to the latest guidelines in a separate commit. Another one down ;-)

**Pull request merged**